mcache/assets/dashboards/memcached_screenboard.json

```
@@ -3,748 +3,1267 @@
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "top"
-      }
-    },
-    {
-      "id": 6114764549509416,
-      "layout": {
-        "x": 66,
-        "y": 79,
-        "width": 34,
-        "height": 16
-      },
-      "definition": {
-        "type": "note",
-        "content": "Due to the way that Memcached assigns memory pages into slabs, it is possible for cache evictions to take place even when the cache is not 100% full. This is why it’s also important to keep an eye on the the eviction rate metric, which is also indicative of the amount of free space in the cache; if the frequency of evictions suddenly increases while fill percent is below 100%, a single slab class may be full.",
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "top"
-      }
-    },
-    {
-      "id": 641803855037284,
-      "layout": {
-        "x": 1,
-        "y": 61,
-        "width": 26,
-        "height": 16
-      },
-      "definition": {
-        "type": "note",
-        "content": "In general, you want to have significantly more cache hits than misses in order for your cache to be effective. Keep an eye on the two metrics, and the ratio between the number of hits and misses; if any of the three start to fall, something is likely wrong.",
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "right"
-      }
-    },
-    {
-      "id": 3420643587887408,
-      "layout": {
-        "x": 174,
-        "y": 7,
-        "width": 47,
-        "height": 36
-      },
-      "definition": {
-        "title": "",
-        "title_size": "16",
-        "title_align": "left",
-        "type": "log_stream",
-        "indexes": [],
-        "query": "service:memcache",
-        "sort": {
-          "column": "time",
-          "order": "desc"
-        },
-        "columns": [
-          "core_host",
-          "core_service"
-        ],
-        "show_date_column": true,
-        "show_message_column": true,
-        "message_display": "expanded-md"
-      }
-    },
+    }
+  ],
+  "template_variables": [
     {
-      "id": 6334176357160004,
-      "layout": {
-        "x": 174,
-        "y": 1,
-        "width": 47,
-        "height": 5
-      },
-      "definition": {
-        "type": "note",
-        "content": "Logs",
-        "background_color": "green",
-        "font_size": "18",
-        "text_align": "center",
-        "show_tick": false,
-        "tick_pos": "50%",
-        "tick_edge": "bottom"
-      }
+      "name": "scope",
+      "default": "*"
     }
   ],
-  "template_variables": [],
-  "layout_type": "free",
+  "layout_type": "ordered",
   "is_read_only": false,
```

```suggestion
  "is_read_only": true,
```

You might want to set this to true in order to prevent edits on OOTB dashboards